src/resource_clients/webhook.ts

```diff
@@ -93,6 +93,7 @@ export interface Webhook {
     lastDispatch: string;
     stats: WebhookStats;
     shouldInterpolateStrings: boolean;
+    isApifyIntegration: boolean;
```

Comment:
```suggestion
    isApifyIntegration?: boolean;
```

Reply:
Thanks for the quick review @valekjo! I assumed it should be optional in `WebhookUpdateData` via `Partial` but required in `Webhook` as it is always part of the response.

All good either way 😄